.ci/ansible/Containerfile.j2

```diff
@@ -2,15 +2,15 @@ FROM {{ ci_base | default("ghcr.io/pulp/pulp-ci-centos:" + pulp_container_tag) }
 
 # Add source directories to container
 {% for item in plugins %}
-{% if item.source.startswith("./") %}
+{% if item.source.startswith("./") or item.ci_requirements | default(false) %}
 ADD {{ item.source }} {{ item.source }}
 {% endif %}
 {% endfor %}
 
 # Install python packages
 # Hacking botocore (https://github.com/boto/botocore/pull/1990)
 
-RUN pip3 install \
```

Comment:
I am "assuming" that if this line-continuation were actually needed, everything would be broken.

Reply:
The `{%-` preamble to the templating block eats the newline (actually the "-" does).